src/inabox/inabox-resources.js

```diff
@@ -62,10 +63,29 @@ export class InaboxResources {
 
     const input = Services.inputFor(this.win);
     input.setupInputModeClasses(ampdoc);
+
+    ampdoc.onVisibilityChanged(() => {
```

Comment:
how is pause/visible handled for `inabox` today?

Reply:
They don't. But they should. Not a very common thing for inabox though comparing to a4a.